samples/mcpclient/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
```

Comment:
is there a reason this can't use the common config files like the other samples?

Reply:
No reason, just a mistake